Packs/CommunityCommonScripts/Scripts/Json2HtmlTable/Json2HtmlTable.yml

```
@@ -2,6 +2,12 @@ args:
 - description: 'The JSON object that should be converted to a HTML table.'
   name: value
   required: true
+- description: 'e.g. pass id=”info-table” or class=”bootstrap-class”/data-* to apply these attributes to the generated table.'
+  name: table-attributes
+  required: false
+- description: 'Custom CSS styling inserted before the table object.'
+  name: custom-styling
```

```suggestion
  name: table_attributes
  required: false
- description: 'Custom CSS styling inserted before the table object.'
  name: custom_styling
```
Hyphens in argument names can cause issues in certain versions.